src/actions/configuration/first-time-configuration-action.php

```diff
@@ -104,7 +123,17 @@ public function set_site_representation( $params ) {
 	 * @return object The response object.
 	 */
 	public function set_social_profiles( $params ) {
-		$failures = $this->social_profiles_helper->set_organization_social_profiles( $params );
+		$failures   = $this->social_profiles_helper->set_organization_social_profiles( $params );
+		$old_values = $this->get_old_values( \array_keys( $this->social_profiles_helper->get_organization_social_profile_fields() ) );
```

Comment:
* I'm noticing that the `get_organization_social_profile_fields` uses but does not check if the filtered return value. Maybe add a safety `(array)` there to ensure this keeps working?
* Also, I see this is the same as for person (with `get_person_social_profile_fields`), but you haven't changed that code in your PR (`set_person_social_profiles` method). Why?

Reply:
The first point has been addressed.
About the second point, `set_person_social_profiles` is dead code, I'm creating a tech debt issue to address that too. 🙂